test/test.js

```diff
@@ -461,9 +461,9 @@ describe('SpecberusWrapper', function () {
       server.getMetadata('webrtc').patentPolicy
     );
 
-    it('should return an error property that has 2 error', function () {
+    it('should return an error property that has 9 errors', function () {
```

Comment:
How come we have so many new errors now?

Reply:
This github action log shows all the errors: https://github.com/w3c/echidna/runs/2304924298?check_suite_focus=true

They are mainly 'response-error' because the **webrtc** document doesn't have a images folder.
```
{
    key: 'response-error',
    extra: {
      url: 'http://localhost:3001/drafts/webrtc/images/icetransportstate.svg',
      status: 404,
      text: 'Not Found',
      referer: 'http://localhost:3001/drafts/webrtc/'
    },
    detailMessage: 'This resource or link found in document (http://localhost:3001/drafts/webrtc/images/icetransportstate.svg) is not responding ok (404 Not Found). Please install it correctly.',
    type: {
      name: 'links.linkchecker',
      section: 'compound',
      rule: 'compoundFilesLocation'
    }
  },
```

Reply:
so in this case, I would rather fix the draft and remove all the external resources. Echidna only checks the number of errors and when you have many, it's hard to debug.